src/python/tests/core/bot/tasks/fuzz_task_test.py

```diff
@@ -1342,7 +1342,10 @@ def test_basic(self):
     engine_impl = mock.Mock()
     engine_impl.name = 'libFuzzer'
     engine_impl.prepare.return_value = engine.FuzzOptions(
-        '/corpus', ['arg'], ['strategy_1', 'strategy_2'])
+        '/corpus', ['arg'], {
+            'strategy_1': 1,
+            'strategy_2': 2
```

Comment:
super nit: maybe change `2` to e.g. `50` to keep it super obvious that it might be any value, not necessarily 1, 2, 3, etc.

Reply:
Done!